ch01.asciidoc

```diff
@@ -19,12 +19,14 @@ Due to the use of onion routing, which is also used in tor, even the nodes that
 
 * Satoshis nSequence number
 * unidirectional payment channels
-* white paper
+* Multiple white papers (Joseph Poon, Thaddeus Dryja - https://lightning.network/lightning-network-paper.pdf) / Christian Decker
 * Milan meeting and creation of BOLTs
 * segwit activation
 * passing of integration tests / mainnet launch
 * Australia Meeting and BOLT 1.1
-* The lightning torch
+* Initial nodes/wallets - eclair, c-lightning etc
+* Reckless - Testing on mainchain.
+* Lightning.place / The lightning torch
```

Comment:
You meant satoshis.place? What was lightning.place?did I miss something?

Reply:
Yeah, I meant satoshis.place.  :)  I think that was the moment a lot of people realized that lightning actually worked and the full power of it.

Reply:
updated!  and I learnt more about how to use Github also...   Thanks :)